src/main/resources/db/cassandra/001_Initialize_db.cql

```diff
@@ -37,64 +41,74 @@ CREATE TABLE IF NOT EXISTS repair_run (
   last_event         text                    ,
   segment_count      int                     ,
   repair_parallelism text
-);
+)
+  WITH compaction = {'class': 'LeveledCompactionStrategy'}
+  AND caching = {'rows_per_partition': 10};
 
 CREATE TABLE IF NOT EXISTS repair_run_by_cluster(
   cluster_name text,
-  id bigint,
+  id           timeuuid,
   PRIMARY KEY(cluster_name, id)
-);
+)
+  WITH compaction = {'class': 'LeveledCompactionStrategy'}
+  AND caching = {'rows_per_partition': 10};
 
 CREATE TABLE IF NOT EXISTS repair_run_by_unit(
-  repair_unit_id bigint,
-  id bigint,
+  repair_unit_id timeuuid,
+  id             timeuuid,
   PRIMARY KEY(repair_unit_id, id)
-);
+)
+  WITH compaction = {'class': 'LeveledCompactionStrategy'}
+  AND caching = {'rows_per_partition': 10};
 
 CREATE TABLE IF NOT EXISTS repair_segment (
-  id               bigint PRIMARY KEY,
-  repair_unit_id   bigint,
-  run_id           bigint,
+  id               timeuuid PRIMARY KEY,
+  repair_unit_id   timeuuid,
+  run_id           timeuuid,
   start_token      varint,
   end_token        varint,
   state            int   ,
   coordinator_host text,
   start_time       timestamp,
   end_time         timestamp,
   fail_count       INT
-);
+)
+  WITH compaction = {'class': 'LeveledCompactionStrategy'}
+  AND caching = {'rows_per_partition': 10};
 
 CREATE TABLE IF NOT EXISTS repair_segment_by_run_id (
-  run_id           bigint,
-  segment_id       bigint,
+  run_id           timeuuid,
+  segment_id       timeuuid,
   PRIMARY KEY(run_id, segment_id)
-);
+)
+  WITH compaction = {'class': 'LeveledCompactionStrategy'}
+  AND caching = {'rows_per_partition': 10};
 
 
 CREATE TABLE IF NOT EXISTS repair_schedule (
-  id                 bigint PRIMARY KEY,
-  repair_unit_id     bigint,
+  id                 timeuuid PRIMARY KEY,
+  repair_unit_id     timeuuid,
   state              text                    ,
   days_between       int                ,
   next_activation    timestamp,
-  run_history        set<bigint>,
+  run_history        set<timeuuid>,
   segment_count      int                     ,
   repair_parallelism text                    ,
   intensity          double                    ,
   creation_time      timestamp,
   owner              text                    ,
   pause_time         timestamp
-);
+)
+  WITH compaction = {'class': 'LeveledCompactionStrategy'}
+  AND caching = {'rows_per_partition': 10};
 
 
 CREATE TABLE IF NOT EXISTS repair_schedule_by_cluster_and_keyspace(
   cluster_name text,
   keyspace_name text,
-  repair_schedule_id bigint,
+  repair_schedule_id timeuuid,
   PRIMARY KEY((cluster_name, keyspace_name), repair_schedule_id)
-);
+)
+  WITH compaction = {'class': 'LeveledCompactionStrategy'}
+  AND caching = {'rows_per_partition': 10};
```

Comment:
This is a significant amount of work (and surface area for bugs) relative to the number of users using the cassandra storage.

I can do it, but i'm wondering if it's really worth the effect. I wouldn't have instead requested to drop the keyspace, and recreate from scratch. Especially given we're a) pre 1.0, and b) cassandra storage is so new.